source/crud/read-operations/cursor.txt

```diff
@@ -0,0 +1,146 @@
+===========================
+Work With Operation Results
+===========================
+
+.. default-domain:: mongodb
+
+Read operations that return multiple documents do not immediately return
+all values matching the query. Instead, these operations rely upon an
+object called a **cursor** that fetches documents in batches. Cursors
+are highly configurable and offer multiple interaction paradigms for
+different use cases.
```

Comment:
Can we give a little intuition as to _why_ we use cursors?

- to work with potentially large result sets that may exceed working memory
- to minimize network bandwidth usage